molgenis-core-ui/src/main/java/org/molgenis/ui/settings/AppDbSettings.java

```diff
@@ -248,10 +248,10 @@ public void setBootstrapTheme(String bootstrapTheme)
 	{
 		// verify that css file exists
 		PathMatchingResourcePatternResolver resolver = new PathMatchingResourcePatternResolver();
-		Resource resource = resolver.getResource("/css/themes/" + bootstrapTheme);
+		Resource resource = resolver.getResource("/css/" + bootstrapTheme);
```

Comment:
This fixes #4337 . We ran into this when we were fiddling with the import order of the theme and the bootstrap css. We fixed the bug cause otherwise we couldn't test if the theme switcher still worked after this PR. (Cause it already wasn't working properly before the PR)